### DISCUSSION TITLE
issue starting metersploit-framework on my kali linux OS

### DISCUSSION THREAD

**Discussion by oracle-venom** [rapid7/metasploit-framework#14366] [General] [answered]
Executing “sudo msfdb init && msfconsole”
[+] Starting database
[+] Creating database user 'msf'
[+] Creating databases 'msf'
[+] Creating databases 'msf_test'
[+] Creating configuration file '/usr/share/metasploit-framework/config/database.yml'
[+] Creating initial database schema
Traceback (most recent call last):
        5: from <internal:gem_prelude>:1:in `<internal:gem_prelude>'
        4: from <internal:gem_prelude>:1:in `require'
        3: from /usr/lib/ruby/2.7.0/rubygems.rb:1424:in `<top (required)>'
        2: from /usr/lib/ruby/2.7.0/rubygems.rb:1424:in `require'
        1: from /usr/lib/ruby/vendor_ruby/rubygems/defaults/operating_system.rb:49:in `<top (required)>'
/usr/lib/ruby/vendor_ruby/rubygems/defaults/operating_system.rb:50:in `<class:Specification>': undefined method `rubyforge_project=' for class `Gem::Specification' (NameError)

**Comment by oracle-venom** [comment]
root@ORACLE:~# msfconsole
Traceback (most recent call last):
        5: from <internal:gem_prelude>:1:in `<internal:gem_prelude>'
        4: from <internal:gem_prelude>:1:in `require'
        3: from /usr/lib/ruby/2.7.0/rubygems.rb:1424:in `<top (required)>'
        2: from /usr/lib/ruby/2.7.0/rubygems.rb:1424:in `require'
        1: from /usr/lib/ruby/vendor_ruby/rubygems/defaults/operating_system.rb:49:in `<top (required)>'
/usr/lib/ruby/vendor_ruby/rubygems/defaults/operating_system.rb:50:in `<class:Specification>': undefined method `rubyforge_project=' for class `Gem::Specification' (NameError)

**Comment by oracle-venom** [comment]
would be glade to have these issue solved.

**Comment by smcintyre-r7** [comment, marked as answer]
I'm going to close this out since this isn't a discussion but rather an issue you'd like to have solved. If you could please reopen it under the Issues category, we can take a closer look at it there. Discussions are still really new for Metasploit, and it, unfortunately, doesn't look like I can transfer this for you.
